VEDA_OS01/models.py

```diff
@@ -428,3 +428,31 @@ def __unicode__(self):
             self.status_email,
             self.file_complete
         )
+
+
+class TranscriptionProviderType(object):
+    """
+    3rd party transcript providers.
+    """
+
+    THREE_PLAY = '3PlayMedia'
+    CIELO24 = 'Cielo24'
+    CHOICES = (
+        (THREE_PLAY, THREE_PLAY),
+        (CIELO24, CIELO24),
+    )
+
+
+class TranscriptionProvider(models.Model):
+    """
+    Model to contain third party transcription service provider preferances.
+    """
+    org = models.CharField('Organization', max_length=50, unique=True)
```

Comment:
can we add help text to tell others that these are organizations from studio / platform, not the institutes in VEDA.